### PR TITLE
Adds the Inquisitor's Clothing

### PR DESCRIPTION
**Pull request by jknpj**
unknown-4

Adding them to the cesspool of adminbus only items so we can use them in the future. Sprites by Frosty.

### CODE REVIEW

**Comment by Dacendeth** [comment]
Would be a cool item to add to the wondrous wardrobe perhaps, that thing needs more variety.

File: code/modules/clothing/suits/miscellaneous.dm [hunk 1]
```diff
@@ -588,3 +588,15 @@
...
+
+
+/obj/item/clothing/suit/inquisitor
+	name = "inquisitor's coat"
+	desc = "This inquisitor attire was made for new recruits, and has excellent straightforward defense. But not nearly enough to allow an ordinary man to stand any real chance against the the wicked."
+	icon_state = "coat-church"
+	item_state = "coat-church"
+	flags = FPRINT
+	body_parts_covered = ARMS|LEGS|FULL_TORSO|IGNORE_INV
+	allowed =  = list(/obj/item/weapon/nullrod, /obj/item/weapon/storage/bible)
```
Comment (Exxion) [inline code/modules/clothing/suits/miscellaneous.dm]:
```
=  =
```

**Comment by Arthurdentist** [comment]
Putting this in the pietyvend would also be pretty cool

**Comment by Irradiation** [comment]
IT SAYS HERE THAT YOU'RE A HERETIC!

**Pull request merged**